admin-node-setup.sh

```diff
@@ -48,3 +48,6 @@ cp $tmp_dir/public.yaml $kube_dir
 cp $tmp_dir/private.yaml $kube_dir
 
 rm -rf $tmp_dir
+
+# switch deprecated --config flag in kubelet
+sed -i -e "s/--config/--kubeconfig/g" /etc/kubernetes/kubelet
```

Comment:
should be moved to `--pod-manifest-path`

Reply:
updated